--- Makefile.in	2018-09-20 11:19:41.115085841 -0400
+++ Makefile.in.noupdate	2018-09-20 11:20:37.423084537 -0400
@@ -833,7 +833,7 @@
 	tar cBf - jemboss | ( cd $(distdir); tar xBf - ; find jemboss -name CVS | xargs rm -rf; find jemboss -name Makefile | xargs rm -rf; find jemboss -name .cvsignore | xargs rm -rf )
 
 install-exec-hook:
-	$(bindir)/embossupdate
+	echo "Skipping embossupdate"
 
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
